config/routes.rb

```diff
@@ -21,9 +21,10 @@
     resources :lines, only: [:new, :create]
   end
   root :to => redirect('/users/sign_in')
-  devise_for :users, skip: [:registrations]
+  devise_for :users, :controllers => { :omniauth_callbacks => "users/omniauth_callbacks" }
+  # devise_for :users, skip: [:registrations]
   as :user do
-    get '/users/edit' => 'devise/registrations#edit', as: 'edit_user_registration'
-    put '/users' => 'devise/registrations#update', as: 'registration'
+    # get '/users/edit' => 'devise/registrations#edit', as: 'edit_user_registration'
```

Comment:
Would like to restore these too. We only need these two routes from registration.